GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -104,6 +105,11 @@ public void CanParseBackwardsDiff()
         [TestCase]
         public void ParsesCaseChangesAsAdds()
         {
+            if (GVFSPlatform.Instance.Constants.CaseSensitiveFileSystem)
```

Comment:
I wonder if there is a way to do this with test categories using the `[]` annotations.

Reply:
Time to find out!

Reply:
be79df24 uses this approach -- thanks!